src/components/chart/common-chart-options.js

```diff
@@ -141,40 +141,6 @@ class CommonChartOptions {
 
     getOptions = () => this.options;
 
-    getMobileOptions = () => {
```

Comment:
@precious-onyenaucheya-ons I might change this so it isn't removed completely - as you will need it for the `tickIntervals` ticket.

Reply:
I've re-added it, but the options object is empty until we need it again.